### ISSUE TITLE
Lower AP results when reproducing V2X-ViT with provided config (OPV2V)

### ISSUE THREAD

**Issue by rb248** [LuckyDush/RCP-Bench#2]
<p>Hi,</p>
<p>First of all, thank you for releasing the code and dataset generation details for <strong>RCP-Bench</strong> — it’s very helpful for my research.</p>
<p>I attempted to reproduce the <strong>V2X-ViT</strong> results on OPV2V using the released config:<br>
<code inline="">HEAL/opencood/hypes_yaml/opv2v/CameraOnly/camera_v2xvit.yaml</code></p>
<h3>My setup</h3>
<ul>
<li>
<p>Environment: Environment
Python: 3.8.10
PyTorch: 1.12.0+cu116
CUDA: 11.6.124
System: Ubuntu 20.04.6 LTS
spconv-cu116: 2.3.6
Hardware: Multi-GPU setup (tested both single and distributed)</p>
</li>
<li>
<p>Training: tested both <strong>single GPU</strong> and <strong>distributed</strong> training modes</p>
</li>
<li>
<p>Dataset: OPV2V, camera-only (scenes listed in attached images below)</p>
</li>
</ul>
<h3>Results</h3>

Method | AP@0.3 | AP@0.5 | AP@0.7
-- | -- | -- | --
Paper (RCP-Bench) | 0.71 | 0.58 | 0.31
Single GPU | 0.58 | 0.46 | 0.23
Distributed | 0.58 | 0.48 | 0.24

<p>As shown, my reproduced AP@0.5 results (~0.46–0.48) are noticeably below the paper (~0.58).</p>
<h3>Dataset</h3>
<p>I have attached images showing the <strong>training</strong> and <strong>testing</strong> scenes I used, listed in order. Could you please confirm whether these match the dataset splits you used? Missing or mismatched scenes could explain the discrepancy.</p>
<p><strong>Image for TRAIN dataset:</strong><br>

<img width="414" height="1314" alt="Image" src="https://github.com/user-attachments/assets/69e5a317-4c46-413a-b241-ac1b715d04ab" />

</p>
<p><strong>Image for TEST dataset:</strong><br>

<img width="692" height="836" alt="Image" src="https://github.com/user-attachments/assets/6ff4db60-517e-48ae-a525-70a466ab93f7" />

</p>
<h3>Notes</h3>
<ul>
<li>
<p>Depth supervision was not used (consistent with the paper).</p>
</li>
<li>
<p>No code modifications were made.</p>
</li>
</ul>
<p>Any guidance or confirmation of dataset splits would be greatly appreciated. Hopefully, this issue can also help others who encounter similar discrepancies.</p>
<p>Thanks again for your time and for sharing this work!</p>

**Comment by LuckyDush** [comment]
Thanks for pointing this out! The dataset split itself is correct. According to the Heal paper, for the camera setting the evaluation range during testing should be [-51.2, 51.2]. After adjusting this metric, the results are consistent with those reported in the paper. I reran the experiments and was able to reproduce them, so I’ll go ahead and close this issue.

**Comment by rb248** [comment]
Yes, thank you for clarifying this; it works for me as well.
